karma.conf.js

```diff
@@ -1,44 +1,37 @@
-const BROWSERS_TO_TEST = ['Chrome']
+const BROWSERS_TO_TEST = ['Chrome'];
 
 module.exports = function(config) {
   config.set({
     basePath: '.',
-    frameworks: ['jasmine', 'closure'],
+    frameworks: ['jasmine'],
     files: [
       // closure library
       {pattern: 'node_modules/google-closure-library/closure/goog/base.js'},
-      {pattern: 'node_modules/google-closure-library/closure/goog/deps.js',
-        included: false, served: false},
       // source files
-      {pattern: 'src/**/*.js', included: false},
-      // tests
-      {pattern: 'test/**_test.js'},
+      {pattern: 'src/**/is_plain_object.js'},
```

Comment:
Does it mean we'll need to list every file individually? or it's only to allow the migration?

Reply:
Unfortunately we do have to list the source files individually. The order matters to the googmodule preprocessor and dependencies need to be processed before the files they are used in.